lib/src/presentation/code_edit_text.dart

```diff
@@ -1,35 +1,60 @@
 import 'package:flutter/material.dart';
+import 'package:kode_view/src/presentation/line_numbers_wrapper.dart';
 import 'package:kode_view/src/presentation/syntax_highlighting_controller.dart';
 import 'package:kode_view/src/presentation/text_selection_options.dart';
 
 class CodeEditText extends StatefulWidget {
   const CodeEditText({
     required this.code,
+
+    // Core functionality
```

Comment:
Even this is a nice segregation, this can lead to problems with linting and formatting, please order params by conceptual groups but remove the comments